src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
-body {
+@import '//fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
+
+* {
```

Comment:
It's a good practice to set `box-sizing: border-box;` using `*` selector

But margin and padding are better to reset for elements you need (where there is a default value) - you can reset all values in a separate file: _reset.scss
font-family will be inherited - so define it once on top level element (html/body)